imagecontent/registries.conf

```diff
@@ -9,17 +9,10 @@
 # and 'registries.block'.
 
 [registries.search]
-registries = ['docker.io', 'registry.fedoraproject.org', 'quay.io', 'registry.access.redhat.com', 'registry.centos.org']
+# registries = ['docker.io', 'registry.fedoraproject.org', 'quay.io', 'registry.access.redhat.com', 'registry.centos.org']
```

Comment:
i'm reasonably certain this is going to cause tests to fail unless we miraculously have fully qualified image specs for all our tests (I did recently go qualify a bunch, but for a long time we had stuff like "FROM centos").

but i guess we'll see.  Anyway you'll definitely want to investigate any aws-builds failures.